src/app/tide/spectrum_preprocess2.cc

```diff
@@ -283,7 +280,18 @@ void ObservedPeakSet::addEvidToResEvMatrix(
       int newResMassBin = bIonMassBin + aaMassBin[curAaMass];
       
       // Find all ion mass bins that match newResMassBin
-      int index = find(ionMassBin.begin(), ionMassBin.end(), newResMassBin) - ionMassBin.begin();
+      vector<int>::iterator ionMassBinItr = find(ionMassBin.begin(), ionMassBin.end(), newResMassBin);
+      // if (ionMassBinItr == ionMassBin.end()){
```

Comment:
Are these commented lines necessary to keep?

Reply:
removed.